dedibox-network/rpn/reference-content/private-networks-overview.mdx

```diff
@@ -36,7 +36,7 @@ Currently, Scaleway offers two variants of the RPN feature, depending on the com
 - **RPNv2** is our most advanced RPN service, providing full VLAN capacity and up to 25GBit/s bandwidth.
   ​
 
-**RPNv1** offers you a static private IP address and allows you to create a private network between your Dedibox machines. Network speeds reach up to 1GBit/s with Business service level.
+**RPNv1** offers you a static private IP address and allows you to create a Private Network between your Dedibox machines. Network speeds reach up to 1GBit/s with Business service level.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
**RPNv1** offers you a static private IP address and allows you to create a private network between your Dedibox machines. Network speeds reach up to 1GBit/s with the Business service level.
```